examples/resources/pingfederate_idp_sts_request_parameters_contract/resource.tf

```diff
@@ -0,0 +1,9 @@
+resource "pingfederate_idp_sts_request_parameters_contract" "requestParametersContract" {
```

Comment:
I wasn't able to find a realistic example for this resource.

Reply:
this is fine